jormungandr/src/rest/v0/logic.rs

```diff
@@ -570,9 +575,7 @@ pub async fn get_diagnostic(context: &Context) -> Result<Diagnostic, Error> {
     Ok(diagnostic_data.clone())
 }
 
-pub async fn get_network_p2p_quarantined(
-    context: &Context,
-) -> Result<Vec<poldercast::Node>, Error> {
+pub async fn get_network_p2p_quarantined(context: &Context) -> Result<Vec<ProfileInfo>, Error> {
```

Comment:
Does this fn and those below need to be public?

Reply:
They are called from `jormungandr/src/rest/v0/handlers.rs`